projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -68,12 +75,18 @@ The component has multiple responsibilities:
 Effects handle external data and interactions, allowing your services to be less stateful and only perform tasks related to external interactions. Next, refactor the component to put the shared movie data in the `Store`. Effects handle the fetching of movie data.
 
 <code-example header="movies-page.component.ts">
+
+import { Component, inject } from '@angular/core';
```

Comment:
If we're importing `inject` here I think it's also good to update the snippet to use inject instead of using the constructor. 
Do you also want to change this, or should we pick this up in a different PR?